### ISSUE TITLE
Abstract structure for how Projecte and its addon mods can be added to the pack

### ISSUE THREAD

**Issue by pollymonosumo93** [dawsontech30000/welcometonightmare-idea-board#4]
In this issue I will only add Abstract structure ideas where you can make your own fundamental structure for an idea from

You make it so there are only Contentweaker items of materials in the Projecte table.

I suggested projecte and its addon mod in this issue #3 , it also explain why I will be making many issues

**Comment by dawsontech30000** [comment]
?

**Comment by pollymonosumo93** [comment]
> ?

There is a issue which is only for me to suggest mods and I will make new issue which talks about how one of the mod could be added and do that to all of the mods I will suggest, That way you could use that issue to keep track of what mod I have suggested and the other issues will help you keep track of ideas how those mod could be added to the pack

**Comment by dawsontech30000** [comment]
alr

**Comment by pollymonosumo93** [comment]
So write here any and all ideas how Projecte and it's addon mods could be added to the pack.

**Comment by pollymonosumo93** [comment]
**I know you don't want the Projecte table be available in the early game and I agree with you there** but the Projecte items can be gotten in other ways then from table which most modpack makers sleep on we can use that to get the Projecte items but have the table only craftable between 87.5% to 94% through the pack.

**Comment by pollymonosumo93** [comment]
What I was going on about in the last post is the World Transmutation, The Collector, The Relay, The Power Flower ETC... those things are rarely focused as much as  the table so in the begging of the pack person could only get Projecte items and items which has emc through World Transmutation, The Collector, The Relay, The Power Flower and such, so that Projecte items can be used as requirement for an items, How does that sound.

**Comment by dawsontech30000** [comment]
still everything to do project e will be gated heavily towards end game

**Comment by pollymonosumo93** [comment]
> still everything to do project e will be gated heavily towards end game

I agree 100% with you there but it would be interesting needing todo the old way of EMC gathering through the pack

**Comment by dawsontech30000** [comment]
got plan will disclose it once get to that chapter still early game right now so lets plan this when near to it

**Comment by pollymonosumo93** [comment]
What about letting people to World transmute grass and stone?

**Comment by dawsontech30000** [comment]
> What about letting people to World transmute grass and stone?

wont need this at all

**Comment by dawsontech30000** [comment]
World transmute will be unless by then have auto crafting

**Comment by pollymonosumo93** [comment]
What about having the old function of Projecte which I don't see anymore where person needs to have burnable emc or klein star because everything done with Projecte cost EMC that way it can be used in early game without overshadowing the other things which is in the early game

**Comment by dawsontech30000** [comment]
already said project e wont be early game

**Comment by dawsontech30000** [comment]
https://www.curseforge.com/minecraft/mc-mods/sky-orchards = https://www.curseforge.com/minecraft/mc-mods/sky-resources = https://www.curseforge.com/minecraft/mc-mods/ex-nihilo-creatio = https://www.curseforge.com/minecraft/mc-mods/ex-compressum

**Comment by pollymonosumo93** [comment]
> I know you don't want the Projecte table be available in the early game and I agree with you there

that is why I said this before I started anything

**Comment by dawsontech30000** [comment]
there four early game mods
before even get in to tech mods too

**Comment by dawsontech30000** [comment]
already spend 2-3days working on four mods

**Comment by dawsontech30000** [comment]
emc going be limited to certin items too fyi point not everything will have it

**Comment by pollymonosumo93** [comment]
oh ok I am just trying to think of a way to get and use all of the fuel variants and all of the star variants and all of the Matters from it in recipes before and after getting Projecte table.

**Comment by pollymonosumo93** [comment]
I am tired of those things are after the table I want the table to be after some of those and before the others so people would need to to have interesting Projecte setups

**Comment by pollymonosumo93** [comment]
> emc going be limited to certin items too fyi point not everything will have it

I undertsand that and I am having that in mind while trying to find interesting way of using it

**Comment by pollymonosumo93** [comment]
Isn't EMC on items fully customizable so can't it be increased

**Comment by SissyFemBoi** [comment]
I recently encountered a modpack titled "Project Overpowered: Skyblock," which introduces a compelling approach to crafting items from ProjectE. This modpack mandates the use of multiblocks throughout the game, requiring players to engage with increasingly complex and challenging recipes from the outset to the conclusion of the game. The multiblocks necessary for crafting become progressively intricate, demanding more elaborate blocks and sophisticated processes to form the multiblock machines. Additionally, the recipes for ProjectE items within these machines also grow more demanding.

For a detailed overview and insights on how ProjectE can be integrated in a balanced manner, I recommend visiting the CurseForge page for "Project Overpowered: Skyblock." This page provides comprehensive information on the modpack's features and implementation strategies: [Project Overpowered: Skyblock on CurseForge](https://www.curseforge.com/minecraft/modpacks/project-overpowered).

Your expertise and insights on refining and balancing the integration of ProjectE within modpacks would be greatly appreciated. Thank you for your time and consideration.

**Comment by pollymonosumo93** [comment]
I recently encountered a modpack titled "Project Overpowered: Skyblock," which introduces a compelling approach to crafting items from ProjectE. This modpack mandates the use of multiblocks throughout the game, requiring players to engage with increasingly complex and challenging recipes from the outset to the conclusion of the game. The multiblocks necessary for crafting become progressively intricate, demanding more elaborate blocks and sophisticated processes to form the multiblock machines. Additionally, the recipes for ProjectE items within these machines also grow more demanding.

For a detailed overview and insights on how ProjectE can be integrated in a balanced manner, I recommend visiting the CurseForge page for "Project Overpowered: Skyblock." This page provides comprehensive information on the modpack's features and implementation strategies: [Project Overpowered: Skyblock on CurseForge](https://www.curseforge.com/minecraft/modpacks/project-overpowered).

Your expertise and insights on refining and balancing the integration of ProjectE within modpacks would be greatly appreciated. Thank you for your time and consideration.
